From: Ben Hutchings <ben@decadent.org.uk>
Date: Tue, 29 Mar 2011 05:09:19 +0100
Subject: [PATCH] parisc: Remove .size directive for flush_alias_page

flush_alias_page is disabled with #if 0, but the .size directive for
it is not.

gas used to accept and ignore .size directives which referred to
undefined symbols.  In binutils 2.21 these are treated as fatal
errors.

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
 arch/parisc/kernel/pacache.S |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/parisc/kernel/pacache.S b/arch/parisc/kernel/pacache.S
index 09b77b2..8332f3e 100644
--- a/arch/parisc/kernel/pacache.S
+++ b/arch/parisc/kernel/pacache.S
@@ -842,6 +842,7 @@ ENTRY(flush_alias_page)
 	.exit
 
 	.procend
+ENDPROC(flush_alias_page)
 #endif
 
 	.export flush_user_dcache_range_asm
-- 
1.7.4.1

